module-rt/src/bootstrap.rs

```diff
@@ -85,6 +89,10 @@ impl<T: UserModule + 'static> FoundryModule for ModuleContext<T> {
         ServiceRef::create_export(port_ as Arc<RwLock<dyn Port>>)
     }
 
+    fn seal(&mut self) {
```

Comment:
I couldn't understand the concept of `seal` here.
Could you explain it to me?

Reply:
https://github.com/CodeChain-io/foundry/commit/8efc4fc83d44e6e5fff95a1b67b0c38afc341e5c
It notifies the module that the bootstrapping is done

Reply:
`seal` in Linker context has a meaning. It will make Linker or Linkable not to link anymore.
However `seal` in module context ambiguous. Please write a comment over the `seal` function or change the name.